### DISCUSSION TITLE
External Resources

### DISCUSSION THREAD

**Discussion by StefanRetief** [codesandbox/sandpack#111] [Q&A] [answered]
I was digging around the repo a little bit and noticed that there is the option to provide the bundler with an `externalResources` property. It does not appear to be exposed, but could it be assumed that this would allow for external libraries (probably ones listed in script tags) to be used along side the bundler?

My use case is specific in that the modules I am trying to use with sandpack are absolutely massive and even using them within codesandbox.io provides varying results. Being able to provide pre-built modules and to reference them would be very beneficial.

**Comment by akserg** [comment]
Hi Stefan

I raised the same concern as the [Feature Request](https://github.com/codesandbox/sandpack/issues/119)

Sergey.

**Comment by danilowoz** [comment, marked as answer]
Hey, for those who are landing in this question, you can find more details of how to use External resources in the documentation.
https://sandpack.codesandbox.io/docs/getting-started/custom-content#static-external-resources
